### PR TITLE
Add functional test for readAsBinaryString

### PR DESCRIPTION
**Pull request by xiuqijix**
Fix issue https://github.com/w3c/web-platform-tests/issues/3374.

@Ms2ger, PTAL.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/23607)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/335514146)

File: FileAPI/reading-data-section/filereader_readAsBinaryString.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<link rel="help" href="https://w3c.github.io/FileAPI/#readAsBinaryString">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+
+async_test(t => {
+  const blob = new Blob(["σ"]);
+  const reader = new FileReader();
+
+  reader.onload = t.step_func(() => {
```
Comment (annevk) [inline FileAPI/reading-data-section/filereader_readAsBinaryString.html]: Nit: you can use `step_func_done()` to avoid having to invoke `done()`.

**Review by annevk — APPROVED** [review]

File: FileAPI/reading-data-section/filereader_readAsBinaryString.html (hunk 1 above)
Comment (kaixinjxq) [reply FileAPI/reading-data-section/filereader_readAsBinaryString.html]: Thanks @annevk, update the nit issue, PTAL.

**Comment by annevk** [comment]
Thanks @kaixinjxq!

**Pull request merged**